house/views.py

```diff
@@ -81,15 +120,17 @@ def house_create(request):
         house_form = HouseCreationForm(request.POST)
         if house_form.is_valid():
             cleaned_data = house_form.cleaned_data
-            House.create_house(user=request.user,
-                               name=cleaned_data['name'],
-                               public=cleaned_data['public'],
-                               parent_profession_1=cleaned_data['parent_profession_1'],
-                               parent_profession_2=cleaned_data['parent_profession_2'],
-                               country=cleaned_data['country'],
-                               city=cleaned_data['city'],
-                               children=cleaned_data['children'],
-                               income=cleaned_data['income'])
+            House.create_house(
+                user=request.user,
+                name=cleaned_data['name'],
+                public=cleaned_data['public'],
+                parent_profession_1=cleaned_data['parent_profession_1'],
+                parent_profession_2=cleaned_data['parent_profession_2'],
+                country=cleaned_data['country'],
+                city=cleaned_data['city'],
+                children=cleaned_data['children'],
+                income=cleaned_data['income'],
+            )
```

Comment:
it seems like this is from your commit and it is just formatting, please remove and add to a different PR.

Reply:
Fixed 👍